docs/en-US/component/cascader.md

```diff
@@ -44,7 +44,7 @@ cascader/last-level
 
 ## Multiple Selection
 
-Set `props.multiple = true` to use multiple selection.
+Add`:props="props"`in tag and set `props : { multiple = true }` to use multiple selection.
 
```

Comment:
I think here might be a little confused for me personally. We should use a code block here to pointing out that what should be done, what should not. E.g.

Do:
```vue
<template>
  <el-cascader :props="props"/>
</template>

<script lang="ts">
export default {
  setup() {
      return {
          props: { // props.
             multiple: true,
          },
      }
  }
}
</script>

```

Don't do

```vue
<template>
  <!--  Object literal binging here is invalid syntax for cascader  -->
  <el-cascader :props="{ multiple: true}"/>
</template>
```

Reply:
@JeremyWuuuuu 
ok
btw，i found there are different languages blocks in [](https://element-plus.org/) but ,i just find en-US in the dev brance of element-plus. how can i change the other language guide